package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-http-auth",
-  "version": "1.2.3",
+  "version": "1.3.0",
```

Comment:
It's not a great practise to change this at feature PR's anyway. Version changes and such should be done by the maintainer at separate commits. Cheers!